src/routes/templates/builder/[slug]/+page.svelte

```diff
@@ -0,0 +1,127 @@
+                          
+                          </svg>
+                    </button>
+                    <button type="submit" class="w-full bg-white text-green-500 border-4 border-green-500 rounded-xl p-2 text-md font-medium 
+                    hover:bg-green-200" on:click={saveTemplate}>
+                    Save Template
+                  </button>
+                </di>
+        </div>
+        
+    </div>
+    <div class="col-span-2 row-span-3 mr-10 border border-custom shadow h-full p-4 rounded-xl">
+        <div class="rounded-md mb-5">
+            <h1 class="block text-md font-medium">Form components</h1>
+            <p>Add elements to customize the form</p>
+        </div>
+  
+        {#each newTemplate.components as component}
+  
+        <div class="bg-white border shadow-md hover:shadow-xl transition duration-250 ease-in-out border-gray-200 rounded-xl px-4 py-3 mb-4">
+            <div class="flex justify-between items-center ">
+                <h2 class="text-lg font-semibold">{component.type}</h2>
+                {#if component.type != "enps-component" && component.type != "department-component"}
+                <div class="flex">
+                    <button 
+                    class="transform hover:scale-150 hover:rotate-180 transition duration-500 ease-in-out text-red-500"
+                    on:click={ () => {
+                        removeComponent(component.id);
+                    }}>
+                        <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" class="w-8 h-8">
+                          <path stroke-linecap="round" stroke-linejoin="round" d="M15 12H9m12 0a9 9 0 11-18 0 9 9 0 0118 0z" />
+                        </svg>
+                    </button>
+                </div>
+                {/if}
+            </div>
+            <div class="">
+                <label for="support" class="text-s mb-2 ">Component <strong>title</strong> or <strong>question</strong></label>
+                <textarea id="support" bind:value={component.custom_text}  placeholder="Provide a text for the " class="border rounded-xl w-full h-16"></textarea>
+            </div>
+        </div>
+        {/each}
+        <div class="flex justify-center mb-3">
+            <button
+                class="m-1 transform hover:scale-150 hover:rotate-90 transition duration-500 ease-in-out text-color"
+                on:click={addComponent}
+              >
+              <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" class="w-10 h-10">
+                <path stroke-linecap="round" stroke-linejoin="round" d="M12 9v6m3-3H9m12 0a9 9 0 11-18 0 9 9 0 0118 0z" />
+              </svg>              
+            </button>
+        </div>
+    </div>
+</div>
+
+
+<style>
+    .custom-color {
+      background-color: #de896e;
+    }
+    .text-color {
+      color: #de896e;;
```

Comment:
You can use the color palette set inside tailwind.config.js as tailwind classes. This colour is in the palette.